mappings/net/minecraft/client/entity/particle/ParticleEntity.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_262 net/minecraft/client/entity/particle/ParticleEntity
```

Comment:
`Particle` or `ParticleEntity`? Other entities (with the exception of the player instance class) don't have the `Entity` suffix.